### PR TITLE
Updating Mitre Attack object

### PR DESCRIPTION
**Pull request by floydtree**
#### Description of changes:
1. Remodeling attack object based on the discussions on Nov 1st, 2023 call and related issue.
2. Updating descriptions across the board, adding examples in descriptions.
3. Deprecating tactics in favor of a non-array attribute - tactic

image

### CODE REVIEW

File: dictionary.json [hunk 1]
```diff
@@ -158,8 +158,8 @@
       "type": "string_t"
     },
     "attacks": {
-      "caption": "Attacks",
-      "description": "An array of attacks associated with an event.",
+      "caption": "MITRE ATT&CK® Details",
```
Comment (zschmerber) [inline dictionary.json:161]: Have we tested special characters in the server? Probably fine as its a string but just thought I would ask.

**Review by zschmerber — APPROVED** [review]
Looks good to me thanks for making this.

File: dictionary.json (hunk 1 above)
Comment (floydtree) [reply dictionary.json:161]: Yeah, have been used in descriptions before. Wouldn't expect such special chars in attribute names though.

**Review by mikeradka — APPROVED** [review]
Looks great, thanks Rajas

**Review by rmouritzen-splunk — APPROVED** [review]

**Pull request merged**